.docker/readeef/scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#/bin/sh
+
+set -x
+set -e
+
+# go get github.com/golang/dep/cmd/dep
+# go get github.com/urandom/readeef/cmd/readeef
+go mod tidy
+pwd
```

Comment:
A call to `go get github.com/urandom/embed/cmd/embed` is needed to fetch the embed command